src/webgpu/shader/execution/shader_io/workgroup_size.spec.ts

```diff
@@ -47,12 +47,14 @@ function checkResults(
   return undefined;
 }
 
-const kSizes = [1, 2, 3, 4, 8, 11, 16, 51, 64, 128, 256];
-
 g.test('workgroup_size')
   .desc(`Test workgroup size is set correctly`)
   .params(u =>
-    u.combine('wgx', kSizes).combine('wgy', kSizes).combine('wgz', kSizes).beginSubcases()
+    u
+      .combine('wgx', [1, 3, 4, 8, 11, 16, 51, 64, 128, 256] as const)
```

Comment:
Should the .combines here be after the .beginSubcases?

Reply:
I decided against that, given that the test is specifically testing the various workgroup sizes. I just reduced the O(n<sup>3</sup>) complexity to something more sensible.

Reply:
sgtm